packages/components/src/internal/components/assay/AssayImportPanels.tsx

```diff
@@ -90,7 +92,6 @@ interface OwnProps {
     allowBulkInsert?: boolean;
     allowBulkUpdate?: boolean;
     fileSizeLimits?: Map<string, FileSizeLimitProps>;
-    maxInsertRows?: number;
```

Comment:
Note that both LKB and LKSM would have been passing in `maxInsertRows={undefined}` after the changes

Reply:
Isn't maxInsertRows used for editable grid, and due to UI limit, we always want to limit editable grid rows to 1000?

Reply:
Ah, gotcha! Re-added